lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+msgid "Error - static memory has been overwritten"
+msgstr "Fel - statiskt minne har skrivits \xefver"
+
+# MSG_CUT_FILAMENT c=16
+#: messages.c:61
+msgid "Cut filament"
+msgstr "Sk\xe1r filament"
+
+# MSG_CUTTER c=9
+#: messages.c:128
+msgid "Cutter"
+msgstr "Sk\xe1rare"
+
+# MSG_MMU_CUTTING_FIL c=18
+#: mmu.cpp:1387
+msgid "Cutting filament"
+msgstr "Sk\xe1r filament"
+
+# MSG_FSENS_NOT_RESPONDING c=20 r=4
+#: ultralcd.cpp:4330
+msgid "ERROR: Filament sensor is not responding, please check connection."
+msgstr "FEL: Filamentsensorn svarar inte, kontrollera anslutningen."
+
+# MSG_DIM c=6
+#: messages.c:159
+msgid "Dim"
+msgstr "Dim"
+
+# MSG_ERROR c=10
+#: messages.c:31
+msgid "ERROR:"
+msgstr "FEL:"
+
+# MSG_SELFTEST_EXTRUDER_FAN_SPEED c=18
+#: ultralcd.cpp:8498
+msgid "Extruder fan:"
+msgstr "Extruderfl\xe1kt:"
+
+# MSG_INFO_EXTRUDER c=18
+#: ultralcd.cpp:2040
+msgid "Extruder info"
+msgstr "Extruder info"
+
+# MSG_EXTRUDER c=17
+#: messages.c:32
+msgid "Extruder"
+msgstr "Extruder"
+
+# MSG_MMU_FAIL_STATS c=18
+#: ultralcd.cpp:6754
+msgid "Fail stats MMU"
+msgstr "Felstatistik MMU"
+
+# MSG_FSENSOR_AUTOLOAD c=13
+#: messages.c:48
+msgid "F. autoload"
+msgstr "F. autoladdn"
+
+# MSG_FAIL_STATS c=18
+#: ultralcd.cpp:6751
+msgid "Fail stats"
+msgstr "Felstatistik"
+
+# MSG_FAN_SPEED c=14
+#: messages.c:36
+msgid "Fan speed"
+msgstr "Fl\xe1ktfart"
+
+# MSG_SELFTEST_FAN c=20
+#: messages.c:91
+msgid "Fan test"
+msgstr "Fl\xe1kttest"
+
+# MSG_FANS_CHECK c=13
+#: messages.c:33
+msgid "Fans check"
+msgstr "Fl\xe1ktcheck"
+
+# MSG_FSENSOR c=12
+#: messages.c:49
+msgid "Fil. sensor"
+msgstr "Fil. sensor"
+
+# MSG_FIL_RUNOUTS c=15
+#: messages.c:34
+msgid "Fil. runouts"
+msgstr "Fil. avbrott"
+
+# MSG_FILAMENT_CLEAN c=20 r=2
+#: messages.c:37
+msgid "Filament extruding & with correct color?"
+msgstr "Extruderas filament med r\xe1tt f\xe1rg?"
+
+# MSG_NOT_LOADED c=19
+#: ultralcd.cpp:2564
+msgid "Filament not loaded"
+msgstr "Filament ej laddat"
+
+# MSG_FILAMENT_SENSOR c=20
+#: messages.c:97
+msgid "Filament sensor"
+msgstr "Filament sensor"
+
+# MSG_FILAMENT_USED c=19
+#: ultralcd.cpp:2713
+msgid "Filament used"
+msgstr "Anv\xe1nt filament"
+
+# MSG_PRINT_TIME c=19
+#: ultralcd.cpp:2714
+msgid "Print time"
+msgstr "Utskriftstid"
+
+# MSG_FS_ACTION c=10
+#: messages.c:151
+msgid "FS Action"
+msgstr "FS aktion"
+
+# MSG_FILE_INCOMPLETE c=20 r=3
+#: ultralcd.cpp:8634
+msgid "File incomplete. Continue anyway?"
+msgstr "Filen \xe1r ofullst\xe1ndig. Forts\xe1tta \xe1nda?"
+
+# MSG_FINISHING_MOVEMENTS c=20
+#: messages.c:45
+msgid "Finishing movements"
+msgstr "Avslutar flyttning"
+
+# MSG_V2_CALIBRATION c=18
+#: messages.c:124
+msgid "First layer cal."
+msgstr "F\xefrstalager kalib."
+
+# MSG_WIZARD_SELFTEST c=20 r=8
+#: ultralcd.cpp:4942
+msgid "First, I will run the selftest to check most common assembly problems."
+msgstr "F\xefrst kommer jag att utf\xefra sj\xe1lvtestet f\xe1r att kontrollera de vanligaste monteringsproblemen."
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.